npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
This file should not be added to the repo, since it's just a temporary error file from npm. You can just delete it entirely :)